### DISCUSSION TITLE
OpenXLA and TF version/schedule.

### DISCUSSION THREAD

**Discussion by kevint324** [openxla/community#9] [xla]
Hi folks

I read the latest sig XLA meeting slides, on `Extracting XLA from TensorFlow A three repos solution` and it says `Expected to deliver: 10/2022`.

So are we expecting to see the delivery of the 3 repo solution on TF version like 2.11 or 2.12?

Thanks
Kevin

**Comment by joker-eph** [comment]
I am not familiar with the TF release schedule, and we're not operating with a particular goal with respect to this. In any case, XLA and TSL will be **vendored** (see https://www.activestate.com/blog/outsourced-dependency-vendoring/ for example if you're not familiar with the term: basically the code will be copied in Tensorflow and there won't be any cross-repo dependencies right now), so what impact to you expect on TensorFlow?

**Comment by kevint324** [comment]
![image](https://user-images.githubusercontent.com/8800468/185917049-0a0987d8-f4ed-4c04-b73c-b574d0a63af0.png)

Thanks for the info about dependency vendoring. 
I thought `tensorflow/compiler/xla` and `tensorflow/stream_executor`  folders would be moved out of TF tree.

As mentions in the slides, `For now we will vendor the XLA and TSL code into tensorflow/third_party to avoid cross-repo synchronization` and `Expected to deliver: 10/2022` 。
My question was about in which TF version will reflect this change.
